packages/storybook/stories/va-notification.stories.jsx

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const notificationDocs = getWebComponentDocs('va-notification');
+
+export default {
+  title: 'Components/Notification',
+  id: 'components/va-notification',
+  argTypes: { 
+    headline: {
+      table: {
+        disable: true,
+      },
+    },     
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  parameters: {
+    componentSubtitle: `va-notification web component`,
+    docs: {
+      page: () => <StoryDocs data={notificationDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  'visible': true,
+  'symbol': 'none',
+  'close-btn-aria-label': 'Close notification',
+  'closeable': true,
+  'has-border': true,
+  'has-close-text': false,
+  'headline': (
+    <h3 part="headline">Notification heading</h3>
+  ),
+  'children': (
+     <p>Notification body</p>
+  ),
+  'href': 'https://www.va.gov/',
+  'text': 'Manage your notifications',
+};
+
+const Template = ({
+  visible,
+  symbol,
+  href,
+  text,
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'has-border': hasBorder,
+  'has-close-text': hasCloseText,
+  headline,
+  children,
+}) => {
+  return (
+    <va-notification
+      visible={visible}
+      symbol={symbol}
+      href={href}
+      text={text}
+      closeable={closeable}
+      has-border={hasBorder}
+      has-close-text={hasCloseText}
+      closeBtnAriaLabel={closeBtnAriaLabel}
+    >
+      {headline}
+      {children}
+    </va-notification>
+  )
+};
+
+const MultipleTemplate = ({
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'has-border': hasBorder,
+}) => {
+  return (
+    <>
+      <va-notification
+        visible="true"
+        symbol="action-required"
+        href="https://www.va.gov/"
+        text="Manage your VA debt"
+        closeable={closeable}
+        has-border={hasBorder}
+        closeBtnAriaLabel={closeBtnAriaLabel}
+        class="vads-u-margin-bottom--1p5"
+      >
+        <h3 part="headline">
```

Comment:
I had another thought about managing the header. I'm assuming (someone from design can confirm) that we want the header text to be styled the same regardless of the header level (ie. font-size). So that might be all the reason we need to bring this in as either a named slot that we style in the component to display a consistent font size for all header levels, or, create two separate props for managing the header; one for the text and one for the header size. 

Examples for either of those approaches:

- Using props for the header in [va-radio](https://design.va.gov/storybook/?path=/docs/components-va-radio--label-header)
- Using slot for the header in [va-accordion](https://design.va.gov/storybook/?path=/docs/components-va-accordion--change-header-level)